### PR TITLE
*/*: EAPI 5->7, indirect and superfluous inherits

### PR DESCRIPTION
**Pull request by thesamesam**

### CODE REVIEW

File: net-analyzer/tleds/tleds-1.05_beta11-r2.ebuild [hunk 1]
```diff
@@ -22,14 +23,16 @@ DEPEND="X? ( x11-libs/libX11 )"
...
 
 src_prepare() {
-	# code patches
-	epatch \
-		"${WORKDIR}"/${MY_P}.patch \
-		"${FILESDIR}"/${P}-gentoo.patch
+	default
+
+	eapply "${WORKDIR}"/${MY_P}.patch
+	eapply "${FILESDIR}"/${P}-gentoo.patch
```
Comment (SoapGentoo) [inline net-analyzer/tleds/tleds-1.05_beta11-r2.ebuild:29]: why no `PATCHES`?

File: net-analyzer/thc-ipv6/thc-ipv6-9999.ebuild [hunk 2]
```diff
@@ -1,46 +1,52 @@
...
 
 src_unpack() {
-	if [[ ${PV} != *9999 ]]; then
-		default_src_unpack
-	else
+	if [[ ${PV} == *9999 ]]; then
 		git-r3_src_unpack
+	else
+		default_src_unpack
 	fi
```
Comment (SoapGentoo) [inline net-analyzer/thc-ipv6/thc-ipv6-9999.ebuild:38]: dont think this conditional is necessary

File: net-analyzer/tcpreplay/tcpreplay-999999.ebuild [hunk 3]
```diff
@@ -49,8 +48,8 @@ src_configure() {
 	# By default it uses static linking. Avoid that, bug 252940
 	econf \
 		$(use_enable debug) \
-		$(use_with pcapnav pcapnav-config /usr/bin/pcapnav-config) \
-		$(use_with tcpdump tcpdump /usr/sbin/tcpdump) \
+		$(use_with pcapnav pcapnav-config "${EPREFIX}"/usr/bin/pcapnav-config) \
```
Comment (SoapGentoo) [inline net-analyzer/tcpreplay/tcpreplay-999999.ebuild:51]: `ESYSROOT`?

File: net-analyzer/synscan/synscan-5.02.ebuild [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
 
 src_prepare() {
 	default
-	mv "$S"/configure.in "$S"/configure.ac || die
-	eautoconf
-}
-
-src_configure() {
-	econf --prefix="${EPREFIX}"/usr
+	mv "${S}"/configure.in "${S}"/configure.ac || die
```
Comment (SoapGentoo) [inline net-analyzer/synscan/synscan-5.02.ebuild:26]: remove `"${S}"/`

File: net-analyzer/snortalog/snortalog-2.4.3-r1.ebuild [hunk 5]
```diff
@@ -24,10 +26,10 @@ RDEPEND="
 	tk? ( dev-perl/Tk )
 "
 
-S=${WORKDIR}
-
 src_prepare() {
-	edos2unix $(find conf/ modules/ -type f) ${PN}.* CHANGES
+	default
+
+	edos2unix $(find conf/ modules/ -type f || die) ${PN}.* CHANGES
```
Comment (SoapGentoo) [inline net-analyzer/snortalog/snortalog-2.4.3-r1.ebuild:32]: while loop?

File: net-analyzer/smokeping/smokeping-2.7.3.ebuild [hunk 6]
```diff
@@ -142,6 +142,6 @@ src_install() {
 }
 
 pkg_postinst() {
-	chown smokeping:smokeping "${ROOT}/var/lib/${PN}"
-	chmod 755 "${ROOT}/var/lib/${PN}"
+	chown smokeping:smokeping "${EROOT}"/var/lib/${PN}
+	chmod 755 "${EROOT}"/var/lib/${PN}
```
Comment (SoapGentoo) [inline net-analyzer/smokeping/smokeping-2.7.3.ebuild:146]: missing `die`?

File: net-analyzer/smokeping/smokeping-2.7.3.ebuild [hunk 7]
```diff
@@ -98,27 +98,27 @@ src_install() {
...
 	sed -i \
 		-e 's:/usr/etc/config:/etc/smokeping/config:' \
-		"${D}/var/www/localhost/smokeping/smokeping.fcgi.dist" || die
+		"${ED}/var/www/localhost/smokeping/smokeping.fcgi.dist" || die
 
 	dodir /var/www/localhost/cgi-bin
-		mv "${D}/var/www/localhost/smokeping/smokeping.fcgi.dist" \
-		"${D}/var/www/localhost/cgi-bin/smokeping.fcgi"
+	mv "${ED}/var/www/localhost/smokeping/smokeping.fcgi.dist" \
+		"${ED}/var/www/localhost/cgi-bin/smokeping.fcgi"
```
Comment (SoapGentoo) [inline net-analyzer/smokeping/smokeping-2.7.3.ebuild:121]: missing `die`

File: net-analyzer/mping/mping-2.01.ebuild [hunk 8]
```diff
@@ -14,6 +13,6 @@ KEYWORDS="amd64 ppc ppc64 sparc x86"
 
 DOCS=( AUTHORS ChangeLog NEWS README TODO )
```
Comment (SoapGentoo) [inline net-analyzer/mping/mping-2.01.ebuild:14]: necessary?

File: net-analyzer/smokeping/smokeping-2.7.3.ebuild (hunk 6 above)
Comment (thesamesam) [reply net-analyzer/smokeping/smokeping-2.7.3.ebuild:146]: I've had people question die in postinst before and I never bothered thinking about it before, let's add it though, people deserve to know when things fail.

File: net-analyzer/tcpreplay/tcpreplay-999999.ebuild (hunk 3 above)
Comment (thesamesam) [reply net-analyzer/tcpreplay/tcpreplay-999999.ebuild:51]: This is tricky - it wants to run the binary, so let's call it "${BROOT}" for -config, and ${ESYSROOT} for tcpdump. Someone can file a bug if we need to do some meddling to pass the right parameters in a cross case.

**Pull request closed without merging**